test/CMakeLists.txt.in

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.2)
+
+project(googletest-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(googletest
+  GIT_REPOSITORY    https://github.com/google/googletest.git
+  GIT_TAG           master
```

Comment:
it might be better to define another tag (in case they change the API), consider release-1.8.1 instead of master (see https://github.com/joaoleal/CppADCodeGen/blob/feature/parameters/test/CMakeLists.txt)

Reply:
I will make a cherry-pick then.